requirements.txt

```diff
@@ -3,7 +3,7 @@ daiquiri
 pipenv
 toml
 delegator.py
-pyyaml
+pyyaml==5.1
```

Comment:
Is there a reason this was explicitly pinned? If there is, it needs to be pinned in `Pipfile` and `Pipfile.lock` as well.

Reply:
Yes, I pinned this because previous versions do not have a function that I'm using.

Reply:
OK, cool. I suggest using `>=` instead, then and please also update the Pipfile and Pipfile.lock